packages/applicant/src/utils/postQuestion.ts

```diff
@@ -160,7 +160,7 @@ export const fieldsStartingWithQuestionIdInBody = (
   questionId: string
 ): [string, string][] => {
   const regex = new RegExp(questionId);
-  return Object.entries(body).filter(([key, value]) => regex.test(key));
+  return Object.entries(body).filter(([key, _value]) => regex.test(key));
```

Comment:
can just do

```suggestion
  return Object.entries(body).filter(([key]) => regex.test(key));
```

to only get 0th index from each array